tableau_utilities/scripts/merge_config.py

```diff
@@ -61,117 +50,46 @@ def merge_2_configs(existing_config, additional_config, debugging_logs=False):
     This assumes that the user is merging the same type of config
 
     Args:
-        existing_config: The current existing config
-        additional_config: The additional config to add
+        existing_config (dict): The current existing config
+        additional_config (dict): The additional config to add
+        debugging_logs (bool): True to print debugging logs
 
     """
-
     for column_name, column_details in additional_config.items():
-
-        if debugging_logs:
-            print(column_name)
-
         # If the column doesn't exist add it
         if column_name not in existing_config:
             if debugging_logs:
                 print("ADDING COLUMN", column_name, column_details)
             existing_config[column_name] = column_details
-
-        elif column_name in existing_config:
+        else:
             if debugging_logs:
                 print('ALTERING COLUMN:', column_name)
-                print('EXISTING COLUMN',  column_name, existing_config[column_name])
-                print('ADDITIONAL COLUMN', column_name, column_details)
-
-            # Set the values from the dictionaries
-            description_current = existing_config[column_name]['description']
-            description_new = column_details['description']
-            folder_current = existing_config[column_name]['folder']
-            folder_new = column_details['folder']
-            persona_current = existing_config[column_name]['persona']
-            persona_new = column_details['persona']
-            datasources_current = existing_config[column_name]['datasources']
-            datasources_new = column_details['datasources']
-
-            # Only used when it's a calculation config
-            if 'calculation' in column_details:
-                calculation_current = existing_config[column_name]['calculation']
-                calculation_new = column_details['calculation']
-
-                if len(calculation_new.strip()) > 0:
-                    if debugging_logs:
-                        print('CHANGING CALCULATION')
-                        print('CALCULATION CURRENT:', calculation_current)
-                        print('CALCULATION NEW:', calculation_new)
-
-                    existing_config[column_name]['calculation'] = calculation_new
+                print('\tEXISTING DETAIL:', existing_config[column_name])
+                print('\tADDITIONAL DETAIL:', column_details)
 
+            # Replace existing config attributes
+            for attribute_name, attribute_value in column_details.items():
+                if str(attribute_value).strip() not in ['', 'None'] and attribute_name != 'datasources':
```

Comment:
**RE the above comment:**
`Revised this comment since I updated the logic a bit`

This line should effectively do the same things as your conditions like
```
if folder_new is not None and len(folder_new.strip()) > 0
```

If `attribute_value` is None or and empty string, this condition will effectively be false. Also, here I am intently not altering the 'datasources' attribute the same as the others, since it's a list and has special logic to be updated (which I handle below).